debug: false
control_rate: 20.0 
torque_on_startup: true
torque_off_on_shutdown: true
dynamixels:
  port_info:
    port_name: /dev/ttyUSB0
    protocol_version: 2
    baud_rate: 3000000
  read_values: # write values are given by the control mode
    read_position: true
    read_velocity: true
    read_effort: true
  control_mode: position
  device_info: # read/write all devices with one packet
    joint_0:
      id: 0
    #joint_1:
      #id: 1
  write_registers:
    joint_0:
      external_port_mod_1: 1 # DO_PP(Digital Output Push-Pull): Use External Port as a digital output port(3.3V level)
    #joint_1:
      #goal_current: 0.05
  #publish_registers:
    #present_temperature: 
      #topic: /dynamixels/temperature
      #frequency: 10 # Hz
      
